src/lib/Scenes/Consignments/v2/State/artworkModel.tsx

```diff
@@ -0,0 +1,85 @@
+import { Action, action, thunk, Thunk } from "easy-peasy"
+import { FormikProps } from "formik"
+import { defaultEnvironment } from "lib/relay/createEnvironment"
+import { commitMutation } from "react-relay"
+import { MyCollectionAddArtworkMutation } from "../Screens/MyCollectionAddArtwork/Mutations/MyCollectionAddArtworkMutation"
+import { MyCollectionEditArtworkMutation } from "../Screens/MyCollectionAddArtwork/Mutations/MyCollectionEditArtworkMutation"
+
+export interface ArtworkFormValues {
+  artist: string
+  title: string
+  year: string
+}
+
+export interface ArtworkModel {
+  formik: FormikProps<any> | null
+  initializeFormik: Action<ArtworkModel, FormikProps<any>>
+
+  addArtwork: Thunk<ArtworkModel, ArtworkFormValues>
+  addArtworkComplete: Action<ArtworkModel>
+  addArtworkError: Action<ArtworkModel>
+
+  editArtwork: Thunk<ArtworkModel, ArtworkFormValues>
+  editArtworkComplete: Action<ArtworkModel>
+  editArtworkError: Action<ArtworkModel>
+}
+
+export const artworkModel: ArtworkModel = {
+  formik: null,
+
+  initializeFormik: action((state, formik) => {
+    state.formik = formik
+  }),
+
+  /**
+   * Add Artwork
+   */
+
+  addArtwork: thunk(async (actions, input) => {
+    try {
+      commitMutation(defaultEnvironment, {
+        query: MyCollectionAddArtworkMutation, // FIXME: Add real mutation once we've completed Gravity API
+        variables: { input },
+        onCompleted: actions.addArtworkComplete,
+        onError: actions.addArtworkError,
+      } as any) // FIXME: any
+    } catch (error) {
+      console.error("Error creating artwork", error)
```

Comment:
```suggestion
      console.error("Error adding artwork", error)
```

very minor and non-blocking but the goal of this change would be parallel language between the error & the action.